src/message.rs

```diff
@@ -33,6 +33,7 @@ pub fn expand(ast: &syn::DeriveInput) -> TokenStream {
 
     quote!{
         mod #dummy_const {
+            use super::*;
             extern crate actix;
 
             impl #impl_generics actix::Message for super::#name #ty_generics #where_clause {
```

Comment:
Since we're going to import everything from `super` let's remove `super::#name` and leave just `#name

Reply:
Agreed. Can you rewrite the commit as such? or should I update the PR?

Thanks for the review.